TaskLayer/MetaMorpheusTask.cs

```diff
@@ -54,6 +54,8 @@ public abstract class MetaMorpheusTask
 
         protected readonly StringBuilder ProseCreatedWhileRunning = new StringBuilder();
 
+        protected string OutputFolder { get; private set; }
```

Comment:
Isn't this in common params and unnecessary?

Reply:
i could not find output folder in common params